package.json

```diff
@@ -8,19 +8,22 @@
   "type": "module",
   "types": "dist/types.d.ts",
   "scripts": {
-    "start": "npm run solid:storybook",
+    "start": "echo \"start has been removed. run dev:<framework> script instead.\" && exit 1",
```

Comment:
Do we want an "Error:" to mimic the `test` message? I like having that to help draw attention to it.